frontend/src/context/mapStateSlice/index.ts

```diff
@@ -41,35 +42,51 @@ const initialState: MapState = {
   boundaryRelationData: {},
 };
 
-// Order layers to keep boundaries and point_data on top. boundaries first.
-export function layerOrdering(a: LayerType, b: LayerType) {
-  if (a.type === 'boundary') {
-    return -1;
-  }
-  if (b.type === 'boundary') {
-    return 1;
+const getTypeOrder = (layer: LayerType) => {
```

Comment:
This seems a bit duplicative of the existing function `getLayerGeometry`. You might want to combine these two?

Reply:
The `getLayerGeometry` is slightly different function I am not sure that those 2 can be combined together. The `getLayerGeometry` function is only used to get the `geometryIconSrc` and nothing more. As I can see in the files. The `getTypeOrder` function is a static utility function that is not exported at all and just tells if a type of a layer is a polygon, also  the `getLayerGeometry ` returns `'point' | 'polygon' | 'raster' | 'unknown'` which are all the known geometry types of a layer. I don't see how those 2 can be combined together.